### PR TITLE
[Core] Adding average HOA lvl in RaidCompositionDetails

### PR DESCRIPTION
**Pull request by Scotsoo**
Implements #2962

### CODE REVIEW

**Comment by Scotsoo** [comment]
First time using react so any feedback is appreciated

**Comment by Sharrq** [comment]
Can you also include a screenshot so whoever is reviewing it can see what it looks like while they are looking at the code?

**Comment by Scotsoo** [comment]
Here you are!
image

File: src/CHANGELOG.js [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
+import { Yajinni, Mamtooth, Zerotorescue, Putro, joshinator, Gebuz, ackwell, emallson, blazyb, Dambroda, Nalhan, Satyric, niseko, Khadaj, Fyruna, Matardarix, jos3p, Aelexe, Chizu, Hartra344, Hordehobbs, Dorixius, Sharrq, Scotsoo } from 'CONTRIBUTORS';
 import ItemLink from 'common/ItemLink';
 import ITEMS from 'common/ITEMS';
 import SPELLS from 'common/SPELLS';
 import SpellLink from 'common/SpellLink';
 import Contributor from 'interface/contributor/Button';
 
 export default [
+  {
+
```
Comment (emallson) [inline src/CHANGELOG.js]: avoid blank lines like this in the future

File: src/interface/report/RaidCompositionDetails.js [hunk 2]
```diff
@@ -10,23 +10,31 @@ class RaidCompositionDetails extends React.PureComponent {
...
-    const { tanks, healers, dps, ranged, ilvl } = this.props;
+    const { tanks, healers, dps, ranged, ilvl, heartLvl } = this.props;
 
     return (
       <div className="raid-composition">
         <div className="bar">
           <div className="panel-body" style={{ padding: 0 }}>
             <div className="flex">
-              <div className="flex-sub icon">
+            <div className="flex-sub icon">
```
Comment (emallson) [inline src/interface/report/RaidCompositionDetails.js]: this should be indented as it was

File: src/CHANGELOG.js (hunk 1 above)
Comment (Scotsoo) [reply src/CHANGELOG.js]: Fixed!

File: src/interface/report/RaidCompositionDetails.js (hunk 2 above)
Comment (Scotsoo) [reply src/interface/report/RaidCompositionDetails.js]: Fixed also!

**Comment by Scotsoo** [comment]
@emallson yeah, I was concerned too with the requests - but I too saw they were cached cause they were originally called in the specific player's screen. I couldn't figure out anywhere else to get the HOA level from - shame it doesn't come back in the player's items on the combatants object where I think we have access to all other bits of gear.

**Pull request merged**